src/google-classroom-extractor/README.md

```diff
@@ -123,3 +123,9 @@ Then:
 1. .ipynb Jupiter Notebook files can be opened directly in
 Visual Studio Code.
 
+## Dev Operations
+
+1. Style check: `poetry run flake8`
+1. Run unit tests: `poetry run pytest`
+1. Run unit tests with code coverage: `poetry run coverage run -m pytest`
+1. View code coverage: `poetry run coverage report`
```

Comment:
_Readme suggestions:_

Users on a corporate network that intercepts TLS/SSL traffic will need to have a copy of the corporate root certificate on file, and then add two variables to the `.env` file (or as plain environment variables). These two will both point to the same certificate file.

```none
REQUESTS_CA_BUNDLE=<absolute path to certificate>
HTTPLIB2_CA_CERTS=<absolute path to certificate>
```

(actually, REQUESTS_CA_BUNDLE might not be needed for google extractor. The second one is *definitely* needed)

Need to clarify where the `service-account.json` and `.env` files go: in `google_classroom_extractor`, not in the parent.

Remove jupyter comments

Comments on the downloaded files.

Reply:
I have now found that setting `HTTPLIB2_CA_CERTS` in the `.env` file does not work - it is not loaded for some reason. I have to set it as an env var at the command line before running `main.py`.

Reply:
Removed jupyter and added TLS/SSL info.

The `service-account.json` and `.env` files should go in the root directory of the project (google-classroom-extractor).  Were you running main.py from there like in the instructions?

Reply:
No, I was unable to run from the _project_ root directory. I only got it working when I changed to the `google-classroom-extractor` directory. Forgot to comment on that. Was probably due to some other issue earlier in the day, which being resolved now, already solved that discrepancy.